packages/worker/string/package.json

```diff
@@ -1,4 +1,6 @@
 {
   "private": true,
-  "main": "../bundle/workerString.ts"
+  "main": "cjs/main.js",
```

Comment:
**💬 suggestion:** ‏to be consistent with how we build other packages, what about building files as `[package]/(cjs|esm)/entries/[entry].[extension]`?
```suggestion
  "main": "../cjs/entries/string.js",
```